CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.12.0...master
+### Fixed
```

Comment:
Also adds `TopNav` to API.

Reply:
- [ ] `### Added` `TopNav` in the changelog